src/config/dynamic-theme-fixes.config

```diff
@@ -12521,16 +12521,6 @@ store.ubi.com
 INVERT
 div.primary-logo
 
-CSS
-.pt_storefront-homepage #main .homepage-slider-wrapper .homepage-custom-slide > a:not(a[href$="/deals"]) .homepage-custom-slide__content .custom-slide-inner h2,
-.pt_storefront-homepage #main .homepage-slider-wrapper .homepage-custom-slide > a:not(a[href$="/deals"]) .homepage-custom-slide__content .custom-slide-inner h3,
-.pt_storefront-homepage #main .homepage-slider-wrapper .homepage-custom-slide a {
-    color: var(--darkreader-neutral-background) !important;
-}
-.pt_storefront-homepage #main .homepage-slider-wrapper .homepage-custom-slide > a[href$="/deals"] .homepage-custom-slide__content .custom-slide-inner h3 {
-    color: var(--darkreader-neutral-text) !important;
-}
-
```

Comment:
Revert this please ;)

Reply:
Done! I don't know how I managed to remove those lines. 😕

Reply:
> Done! I don't know how I managed to remove those lines. confused

It was a recent fix, most likely they didn't exist in your version when you edited the file. It can happen.